.github/workflows/main.yml

```diff
@@ -52,7 +58,7 @@ jobs:
         with:
           fetch-depth: 300
       - name: Fetch tags
-        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+        run: git fetch --depth=300 origin +refs/tags/*:refs/tags/*
```

Comment:
300 is not needed on the test docs runner. It should just verify the docs config and files. The final build is irrelevant.